tools/BUILD

```diff
@@ -106,7 +106,7 @@ tfrt_cc_library(
         "@tf_runtime//:basic_kernels_alwayslink",
         "@tf_runtime//:core_runtime_alwayslink",
         "@tf_runtime//:data_alwayslink",
-        "@tf_runtime//:io_alwayslink",
+        "@tf_runtime//:io",
```

Comment:
Why is this change needed?

Reply:
To account for the changes made to lib/io (`static_registration.cc` has been removed)

Reply:
I don't think this will quite work. The `io_alwayslink` target uses `alwayslink = True`, which is required for the static registration but we also want to have the normal `io` without registration.

Would it work to have the posix and windows registration functions use the same name (exactly one of them is included in the srcs), and then call that from static_registration.cc?

I'm getting back to you on this because changing this particular line breaks the copybara import: there are google internal targets just below that are stripped/inserted during export/import, and the way this is setup prevents nearby changes. This is something we will need to fix on our side, but here I believe this line should stay unchanged.

Reply:
I was afraid this may be the case. Putting things back into `static_registration.cc` is not too bad though, it should just require a platform `#ifdef`. I'll try to get these changes ready asap.

Reply:
Done. 

> Would it work to have the posix and windows registration functions use the same name (exactly one of the two definitions is included in the srcs), and then call that from static_registration.cc?

This idea worked out nicely, thanks.